source/github-helpers/api.ts

```diff
@@ -174,6 +182,10 @@ export const v4 = mem(async (
 
 	query = query.replace('repository() {', () => `repository(owner: "${getRepo()!.owner}", name: "${getRepo()!.name}") {`);
 
+	void log(`{
```

Comment:
Let’s also add the `query` word to this so it can actually be copy-pasted

Reply:
Where is that required? In the explorer it not.